subprojects/core/src/integTest/groovy/org/gradle/api/services/BuildServiceIntegrationTest.groovy

```diff
@@ -653,22 +834,11 @@ class BuildServiceIntegrationTest extends AbstractIntegrationSpec {
         dest.file
     }
 
+
```

Comment:
💄 Is the additional empty line intentional?

Reply:
Sigh... I keep missing them in IntelliJ due its virtual formatting feature (to give room for decorations). I often delete seemingly extra empty lines just to realize they are not actually extra lines as well...